bindings/python/crocoddyl/core/solvers/ddp.cpp

```diff
@@ -83,6 +83,8 @@ void exposeSolverDDP() {
            "It rollouts the action model given the computed policy (feedforward terns and feedback\n"
            "gains) by the backwardPass. We can define different step lengths\n"
            ":param stepLength: applied step length (<= 1. and >= 0.)")
+       .def("computeKp", &SolverDDP::computeKp, bp::args("self", "t","dWp","dKp"),
```

Comment:
Again. This should not be a part of this PR. It is a part of your devel++ branch. 

The idea of target displacement at knot t is not something generic that you could introduce in the library.

Reply:
Please remove computeKp from this PR as well